Package.resolved

```diff
@@ -24,8 +24,8 @@
         "repositoryURL": "https://github.com/nerdsupremacist/AssociatedTypeRequirementsKit.git",
         "state": {
           "branch": null,
-          "revision": "d861e6dd41ad0c1096a9a730611fa6a8cc307b0c",
-          "version": "0.3.1"
+          "revision": "e71ec9822ce06b62011e36272e8014b828916d0b",
+          "version": "0.3.0"
```

Comment:
you might not want to revert to 0.3.0
0.3.1 (and 0.3.2) contain important fixes.

Reply:
good catch. fixed